### ISSUE TITLE
Can't save CountryField with None value

### ISSUE THREAD

**Issue by sairon** [SmileyChris/django-countries#47]
I have a model with a CountryField that has allowed null values. However, when creating a new instance of that model, an empty string is saved into database instead, causing unique constraints to fail. It seems that problem is in `get_prep_value` method, because it receives a Country object with code attribute set to None instead of primitive `None` value.

Example model:

``` python
class Failing(models.Model):
    country = CountryField(null=True, unique=True)
    count = models.PositiveIntegerField(default=0)
```

Failing code:

``` python
a, created1 = Failing.objects.get_or_create(country=None)
b, created2 = Failing.objects.get_or_create(country=None)

assert a == b
assert created1 is True
assert created2 is False
```

**Comment by Ashish1206** [comment]
I am using django-countries 3.0 and getting `country(code=u'') is not JSON serializable`  when I set the country as none. Isn't this issue fixed in 3.3 ?

**Comment by iMerica** [comment]
Issue persists. Please Re-open.
